litr/src/main/java/com/linkedin/android/litr/resample/DownsampleAudioResampler.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2021 natario1 Transcoder
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// from: https://github.com/natario1/Transcoder/blob/main/lib/src/main/java/com/otaliastudios/transcoder/resample/DownsampleAudioResampler.java
+// modified: changed the signature of resample() method
+// modified: small modification for "outputBuffer.put(inputBuffer.get());" logic
+// modified: small modification for "inputBuffer.position(inputBuffer.position() + 1);" logic
+package com.linkedin.android.litr.resample;
+
+import android.media.MediaFormat;
+import androidx.annotation.NonNull;
+import java.nio.ByteBuffer;
+
+/**
+ * An {@link AudioResampler} that downsamples from a higher sample rate to a lower sample rate.
+ */
+public class DownsampleAudioResampler implements AudioResampler {
+
+    private static float ratio(int remaining, int all) {
```

Comment:
I guess this was original static and got copied there? Can we make this not static?